folio-backend-common/pom.xml

```diff
@@ -14,13 +14,19 @@
 
   <properties>
     <sonar.exclusions>
+      src/main/java/**/NoopHostnameVerifier.java,
       src/main/java/**/domain/model/**/*
     </sonar.exclusions>
     <feign-okhttp.version>13.3</feign-okhttp.version>
     <semver4j.version>5.3.0</semver4j.version>
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Suggestion: move everything related to TLS from `folio-backed-common` to `folio-tls-utils`

Reply:
Done